### PR TITLE
RavenDB-17158 increased timeout

### PR DESCRIPTION
**Pull request by ppekrol**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17158

### Additional description

- Increased the timeout for AWS, Azure and GCP libraries so huge backups will not be timing out

### Type of change

- Regression bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- Documentation update is needed. Issue was marked with Documentation Required tag.

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs [hunk 1]
```diff
@@ -50,7 +51,12 @@ public RavenAwsGlacierClient(GlacierSettings glacierSettings, Progress progress
             else
                 credentials = new SessionAWSCredentials(glacierSettings.AwsAccessKey, glacierSettings.AwsSecretKey, glacierSettings.AwsSessionToken);
 
-            _client = new AmazonGlacierClient(credentials, region);
+            _client = new AmazonGlacierClient(credentials, new AmazonGlacierConfig
+            {
+                RegionEndpoint = region,
+                Timeout = RequestExecutor.GlobalHttpClientTimeout
```
Comment (ayende) [inline src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs]: How is the user supposed to be able to configure that? Do we allow that?

Comment (ayende) [reply src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs]: What if they need more? This is a `static readonly` value, and we might need a configurable option.

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsGlacierClient.cs]: We never allowed that, but I've added a configuration option now.

**Review by ayende — APPROVED** [review]

**Review by arekpalinski — APPROVED** [review]

**Comment by ppekrol** [comment]
Renamed the option to 'Backup.CloudStorageOperationTimeoutInMin' because this is for all the operations

**Pull request merged**